.github/workflows/pint.yml

```diff
@@ -1,6 +1,10 @@
 name: PHP linting (Pint)
 
-on: [push]
+on:
+  push:
+    branches: [main]
```

Comment:
Are you sure? We want to run pint before reaching the main branch. It should be run on pull requests only, shouldn't it?